main_test.go

```diff
@@ -100,3 +100,47 @@ func TestDiscoveryWithMockSources(t *testing.T) {
 
 	})
 }
+
+func TestAddLabels(t *testing.T) {
+
+	Convey("When adding labels", t, func() {
+		helper := k8sHelpers{}
+		labels := Labels{}
+		n := &api.Node{
+			ObjectMeta: api.ObjectMeta{
+				Labels: map[string]string{},
+			},
+		}
+
+		Convey("If no labels are passed", func() {
+			helper.AddLabels(n, labels)
+
+			Convey("None should be added", func() {
+				So(len(n.Labels), ShouldEqual, 0)
+			})
+		})
+
+		Convey("They should be added to the node.Labels", func() {
+			test1 := Namespace + ".test1"
+			labels[test1] = "true"
+			helper.AddLabels(n, labels)
+			So(n.Labels, ShouldContainKey, test1)
+
+			Convey("And they should be overwritten by subsequent calls", func() {
+				labels[test1] = "false"
```

Comment:
Right now we don't support key values other than `true` or updates to already exisiting labels (other than removal after this PR gets merged).  Can we delete this test case?